### DISCUSSION TITLE
Does it work for Italy?

### DISCUSSION THREAD

**Discussion by Razer0123** [BottlecapDave/HomeAssistant-OctopusEnergy#593] [Q&A]
Hello, wondering if anyone knows if this works for Italy too

**Comment by BottlecapDave** [comment]
Hello. I don't believe Italy uses the same API as the UK branch, so this probably won't work. The best way to find out is to try?

**Comment by webysther** [comment]
Anyone tested on this? I think I will switch to the branch of italy and after test I will let you know
